lib/comfortable_mexican_sofa/seeds.rb

```diff
@@ -78,8 +79,9 @@ def initialize(from, to = from)
       self.site = Comfy::Cms::Site.where(identifier: from).first!
     end
 
-    def export!
-      %w[Layout Page Snippet File].each do |klass|
+    def export!(klasses = nil)
+      klasses = klasses || %w[Layout Page Snippet File]
+      klasses.each do |klass|
```

Comment:
ditto. Also I'd define the list of classes in the constant. Like: `SEED_CLASSES = %w[Layout Page Snippet File]`